README.md

```diff
@@ -12,6 +12,7 @@ Both Social Team members and Node.js project collaborators submitting PRs should
 - We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
 - Where possible, we try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
   - As an example, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+  - As an exception, in the case where important or potentially negative content is being shared, using the main account's voice is preferable to both reduce potential negative impact on the indivduals we'd be amplifying and increase the message's signal to end-users (seeing someone you don't know vs. seeing the account have different levels of impact).
```

Comment:
```suggestion
  - As an exception, in the case where important or potentially negative content is being shared, using the main account's voice is preferable to both reduce potential negative impact on the indivduals we'd be amplifying and increase the message's signal to end-users (seeing someone you don't know vs. seeing the account have different levels of impact). The current list of regularly occuring content to which this applies includes:
    * security releases
```

Reply:
This is not my lane, but since TSC was pinged: I think trying to maintain a list of content to which this applies is an anti-pattern. (The use of the word "current" or "currently" tends to be a documentation smell, in my opinion.)

I think you can also eliminate all the stuff about "negative" and simply state that there are messages (such as announcements of security releases) where it is preferable that it be sent in the Node.js account voice or whatever.

I'll see if GitHub will let me make a multi-line suggestion...

Reply:
I agree with not maintaining a list - social can be rather scattershot in terms of "situations that come up" and this might very well apply to situations that aren't listed.